spec/modules/batch_spec.rb

```diff
@@ -5,8 +5,8 @@
 
   context 'adding' do
     it 'must enqueue unbatched worker' do
-       RegularWorker.perform_async('bar')
-       expect(RegularWorker).to have_enqueued_job('bar')
+      RegularWorker.perform_async('bar')
+      expect(RegularWorker).to have_enqueued_job('bar')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.